### DISCUSSION TITLE
<a> tag in electron

### DISCUSSION THREAD

**Discussion by 1377sayaba** [quasarframework/quasar#13452] [CLI - Electron mode]
i used <a> tag in my project when i click on it look like crashed and all of screen became White
video  https://aparat.com/v/0bJR1 
can i use <a> tag for electron ?

**Comment by MilosPaunovic** [comment]
Why not using `router-link` instead?

**Reply by 1377sayaba** [reply]
Can you send a documention link
Thanks

**Reply by MilosPaunovic** [reply]
You're gonna have to search `VueRouter` documentation yourself.
